src/views/OverlayView.svelte

```diff
@@ -70,7 +70,5 @@
       </div>
     </div>
   {/if}
-  {#if $state.offerReconnect && isInputPatternValid()}
-    <ReconnectPrompt />
-  {/if}
+  <ReconnectPrompt isOpen={true} />
```

Comment:
Presume this change is unintentional and for local testing.

Reply:
Ahh yes, good spot

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/190/commits/3be043111a52bf43292b634839e6b433f4599c6e

Reply:
This should be 
`<ReconnectPrompt isOpen={$state.offerReconnect && isInputPatternValid()} />`
without the if statement.

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/190/commits/99bae74d9fa1d7b648b66e5fe5b55669b70f257b